app/javascript/email_pension/EmailPensionView.js

```diff
@@ -184,39 +185,32 @@ class EmailPensionView extends Component {
 }
 
 type EmailPensionType = {
-  emailBody: string,
-  emailHeader: string,
-  emailFooter: string,
-  emailSubject: string,
-  country: string,
   email: string,
   name: string,
   isSubmitting: boolean,
-  to: string,
-  fund: string,
   fundContact: string,
   fundEmail: string,
+  fund: string,
+  fundId: string,
+  country: string,
 };
```

Comment:
Just a note that this file isn't actually being type checked! If you want it to be checked by flow, just add the `// @flow` comment at the top of the file.